checks/apps/pytorch/mlperf_storage_ce.py

```diff
@@ -71,12 +71,15 @@ class MLperfStorageCE(rfm.RunOnlyRegressionTest, ContainerEngineMixin):
                        'v1.0-mpi_4.2.1')
     valid_systems = ['+nvgpu +ce']
     valid_prog_environs = ['builtin']
+    tags = {'production', 'ce'}
```

Comment:
```suggestion
    tags = {'production', 'ce', 'maintenance'}
```
This test could be also useful post maintenance, right?